x-pack/test/security_solution_api_integration/test_suites/entity_analytics/default_license/risk_engine/asset_criticality_privileges.ts

```diff
@@ -0,0 +1,199 @@
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+const USER_PASSWORD = 'changeme';
+const ROLES = [
+  {
+    name: 'security_feature_read',
+    privileges: {
+      kibana: [
+        {
+          feature: {
+            siem: ['read'],
+          },
+          spaces: ['default'],
+        },
+      ],
+    },
+  },
+  {
+    name: 'asset_criticality_index_read',
+    privileges: {
+      elasticsearch: {
+        indices: [
+          {
+            names: ['.asset-criticality.asset-criticality-*'],
+            privileges: ['read'],
+          },
+        ],
+      },
+    },
+  },
+  {
+    name: 'asset_criticality_index_write',
+    privileges: {
+      elasticsearch: {
+        indices: [
+          {
+            names: ['.asset-criticality.asset-criticality-*'],
+            privileges: ['write'],
+          },
+        ],
+      },
+    },
+  },
+];
+
+const ALL_ROLE_NAMES = ROLES.map((role) => role.name);
+
+const allRolesExcept = (role: string) => ALL_ROLE_NAMES.filter((r) => r !== role);
+
+const USERNAME_TO_ROLES = {
+  no_asset_criticality_index_read: allRolesExcept('asset_criticality_index_read'),
+  no_asset_criticality_index_write: allRolesExcept('asset_criticality_index_write'),
+  all: ALL_ROLE_NAMES,
+};
+
+export default ({ getService }: FtrProviderContext) => {
+  describe('@ess privileges_apis', () => {
+    const supertestWithoutAuth = getService('supertestWithoutAuth');
+    const assetCriticalityRoutesNoAuth =
+      assetCriticalityRouteHelpersFactoryNoAuth(supertestWithoutAuth);
+    const security = getService('security');
+
+    const createRole = async ({ name, privileges }: { name: string; privileges: any }) => {
+      return await security.role.create(name, privileges);
+    };
+
+    const createUser = async ({
+      username,
+      password,
+      roles,
+    }: {
+      username: string;
+      password: string;
+      roles: string[];
+    }) => {
+      return await security.user.create(username, {
+        password,
+        roles,
+        full_name: username.replace('_', ' '),
+        email: `${username}@elastic.co`,
+      });
+    };
+
+    async function createPrivilegeTestUsers() {
+      const rolePromises = ROLES.map((role) => createRole(role));
+
+      await Promise.all(rolePromises);
+      const userPromises = Object.entries(USERNAME_TO_ROLES).map(([username, roles]) =>
+        createUser({ username, roles, password: USER_PASSWORD })
+      );
+
+      return Promise.all(userPromises);
+    }
+
+    const getPrivilegesForUsername = async (username: string) =>
+      assetCriticalityRoutesNoAuth.privilegesForUser({
+        username,
+        password: USER_PASSWORD,
+      });
+    before(async () => {
+      await createPrivilegeTestUsers();
+    });
+
+    describe('Asset Criticality privileges API', () => {
+      it('should return has_all_required true for user with all risk engine privileges', async () => {
```

Comment:
Nit: as I mentioned in standup, I'd prefer if test writers were a little more confident in their assertions:

```suggestion
      it('returns has_all_required true for user with all risk engine privileges', async () => {
```